dataeng/resources/answer-distribution.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+if [ -z "$NUM_REDUCE_TASKS" ]; then
+    NUM_REDUCE_TASKS=$(( $NUM_TASK_CAPACITY * 2 ))
+fi
+NUM_REDUCE_TASKS=$(( $NUM_REDUCE_TASKS * 2 ))
```

Comment:
We don't have to do this now, but at some point we should really fix this logic so that we figure out the multiplier for NUM_TASK_CAPACITY to determine NUM_REDUCE_TASKS, depending on the instance type.  Rather than hardcoding it here like this.